x-pack/plugins/observability_solution/inventory/e2e/cypress/e2e/home.cy.ts

```diff
@@ -134,7 +134,7 @@ describe('Home page', () => {
         );
       });
 
-      it('Navigates to discover with kuery filter', () => {
+      it.skip('Navigates to discover with kuery filter', () => {
```

Comment:
I am adding an extra link to discover in my [draft](https://github.com/elastic/kibana/pull/199306) and extracted the discover link logic in a new hook. We will probably still have conflicts after because of the change in the search context (we are both adding the data view but here we have more changes) even if the discover link is not changed here 😞 But I can fix the issue with the filters once this PR is merged (depending on the timing it will be either the draft or a new PR)